com.microsoft.mrtk.core/MRTKAudioMixer.mixer

```diff
@@ -0,0 +1,153 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
It looks like this is at the top-level directory of the core package, is that intentional?

Reply:
mostly because i did not want to create a folder for a single (plus meta) file.

Reply:
moved to the Configuration folder